examples/forward/plot_frequency_interference.py

```diff
@@ -0,0 +1,164 @@
+
+lh, rh = fwd['src']
+seed_vert = lh['rr'][stc.lh_vertno[99]]  # reference, zero phase-offset
+phase_shift = 1 / stc.sfreq  # 1 ms shifted per mm distant
+max_dist = 30  # distance in mm of nearly in-phase sources
+
+
+def euclidean_dist(vert0, vert1):
+    return np.sum((vert1 - vert0) ** 2) ** 0.5
+
+
+fig, axes = plt.subplots(1, freqs.size, figsize=(10, 5))
+fig.suptitle('Source Time Courses')
+stcs = dict()  # make source time courses for each frequency sine wave
+for ax, freq in zip(axes, freqs):
+    for i in range(stc.data.shape[0]):
+        vert = lh['rr'][stc.lh_vertno[i]] if i < stc.lh_vertno.size else \
+            rh['rr'][stc.rh_vertno[i - stc.lh_vertno.size]]
+        dist = euclidean_dist(seed_vert, vert) * 1e3  # m -> mm
+        stc.data[i] = np.sin(
+            2. * np.pi * freq * (stc.times + phase_shift * dist)) * 1e-10 \
+            if dist < max_dist else 0  # if too large distance, no oscillation
+
+    stcs[freq] = stc.copy()
+    psds, freqs_psd = psd_array_multitaper(stc.data, stc.sfreq,
+                                           fmin=2, fmax=55)
+    ax.plot(freqs_psd, psds[0])
+    ax.set_title(f'{freq} Hz')
+
+
+axes[0].set_ylabel('Power')
+axes[freqs.size // 2].set_xlabel('Frequency (Hz)')
+
+fig.tight_layout()
+fig.show()
+
+##############################################################################
+# Compute evoked with forward model
+# ---------------------------------
+#
+# Now we can simulate evoked with our pure sine wave source time course
+# with known frequency peaks. We see that the lower the frequency, the greater
+# the power. This is because the same millisecond differences in phase
+# offset results in greater descructive interference for higher frequencies
+# because, for higher frequencies, the period is shorter so the same amount of
+# time is a greater proportion of the period and thus the signals are more
+# offset in phase. This causes greater descructive interference.
+
+fig, axes_all = plt.subplots(3, freqs.size, figsize=(12, 8))
+fig.suptitle('1/f Power Decay in Simulated Evoked')
+
+for axes, freq in zip(axes_all.T, freqs):
+    evoked_sim = simulate_evoked(fwd, stcs[freq], evoked.info,
+                                 cov=None, nave=np.inf)
+    psds, freqs_psd = psd_array_multitaper(
+        evoked_sim.data, stc.sfreq, fmin=2, fmax=60)
+
+    axes[0].plot(freqs_psd, psds[mne.pick_types(evoked_sim.info, meg='mag')].T)
+    axes[0].set_title(f'mag {freq} Hz')
+    axes[0].set_ylim([0, 1e-26])
+    axes[1].plot(
+        freqs_psd, psds[mne.pick_types(evoked_sim.info, meg='grad')].T)
+    axes[1].set_title(f'grad {freq} Hz')
+    axes[1].set_ylim([0, 1e-23])
+    axes[2].plot(freqs_psd, psds[mne.pick_types(evoked_sim.info, eeg=True)].T)
+    axes[2].set_title(f'eeg {freq} Hz')
+    axes[2].set_ylim([0, 1e-11])
+
+
+for ax in axes_all.flatten():
+    ax.set_xlim([5, 50])
+
+
+axes_all[1, 0].set_ylabel('Power')
+axes_all[0, freqs.size // 2].set_xlabel('Frequency (Hz)')
```

Comment:
it shows that the decay is even stronger on the EEG so the skull does increase the interference no?

Reply:
Hmmm that does seem like the logical conclusion but it also isn't the main effect, but maybe worth being a bit more conservative with the language